lib/page/assets/lock/LockDetailPage.dart

```diff
@@ -0,0 +1,146 @@
+
+  static final String route = '/assets/lock/detail';
+  final AppStore store;
+
+  @override
+  _DetailPageState createState() => _DetailPageState(store);
+}
+
+class _DetailPageState extends State<LockDetailPage> {
+  _DetailPageState(this.store);
+
+  final AppStore store;
+
+  @override
+  Widget build(BuildContext context) {
+    var dic = I18n.of(context).assets;
+    
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(dic['lock.tokens'])
+      ),
+      body: SafeArea(
+        child: Builder(
+          builder: (BuildContext context) {
+          return ListView(
+            shrinkWrap: true, 
+            padding: const EdgeInsets.all(20),
+            children: <Widget>[
+              Text(
+                dic['transaction.message'],
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 10),
+                child: Text(
+                  dic['lock.transaction.instruction'],
+                  textAlign: TextAlign.center,
+                ),
+              ),
+              subTitle(dic['formula']),
+              Row(
+                children: <Widget>[
+                  formulaInput(
+                    lable: dic['lock.duration']
+                  ),
+                  Text(' , '),
+                  Expanded(
+                    child: Column(
+                      children: <Widget>[
+                        TextField(
+                          textAlign: TextAlign.center,
+                        ),
+                        Container(
+                          padding: const EdgeInsets.all(5),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: <Widget>[
+                            Text(
+                              '${dic["signal"]},${dic["lock"]}',
+                              style: TextStyle(
+                                fontSize: 10
+                              )
+                            ),
+                            Icon(Icons.arrow_drop_down)
+                          ]),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Text(' , '),
+                  formulaInput(
+                    lable: dic['amount.tokens']
+                  ),
+                ]
+              ),
+              subTitle(
+                dic['your.transaction'],
+              ),
+              ListTile(
+                contentPadding: const EdgeInsets.symmetric(vertical: 10),
+                title: Container(
+                  padding: const EdgeInsets.all(5),
+                  color: Colors.grey[200],
+                  child: Text(
+                    '12,${dic["lock"]},50',
```

Comment:
On this 1st "Lock Tokens" page, instead of hard-coding `12,Lock,50`, please populate this value based on what input field and selection box values are entered by the user. Only hard-code "Lock", so it is `??,Lock,??`, where ?? is populated based on input field and selection box values entered by the user.